models/rubicon/rubicon_trades.sql

```diff
@@ -9,7 +10,8 @@
 
 {% set rubi_models = [
 ref('rubicon_optimism_trades'),
-ref('rubicon_arbitrum_trades')
+ref('rubicon_arbitrum_trades'), 
```

Comment:
This model needs to be migrated first.
By `migrated` means it has `tags = ['dunesql']` in its config block and can be detected by the dbt.

Reply:
looks like this spell is still missing?
@denverbaumgartner

Reply:
hey all, just wanted to verify, this spell does appear to be migrated. you can find it in `main` [here](https://github.com/duneanalytics/spellbook/blob/9f038322c8129b7330629d83587e0a009c950b50/models/rubicon/arbitrum/rubicon_arbitrum_trades.sql#L2C2-L2C2) with the appropriate `dunesql` tag and `block_month` partition